### PR TITLE
signin-activity-update-050223

### PR DESCRIPTION
**Pull request by shlipsey3**

### CODE REVIEW

**Comment by shlipsey3** [comment]
ready for content review

**Comment by shlipsey3** [comment]
#sign-off

File: api-reference/beta/resources/user.md [hunk 1]
```diff
@@ -240,7 +240,7 @@ This resource supports:
...
+| signInActivity | [signInActivity](signinactivity.md) | Get the last signed-in date and request ID of the sign-in for a given user. Read-only.<br><br>Returned only on `$select`. Supports `$filter` (`eq`, `ne`, `not`, `ge`, `le`) *but* not with any other filterable properties. <br><br>**Note:** <br/><li> Details for this property require an Azure AD Premium P1/P2 license and the **AuditLog.Read.All** permission.<li>You must specify `$select=signInActivity` or `$filter=signInActivity` while [listing users](../api/user-list.md), as the signInActivity property is not returned by default.<li>This property is not returned for a user who has never signed in or last signed in before April 2020.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/user.md:243]: The text is already covered by the standard statement "Returned only on `$select`".
```suggestion
| signInActivity | [signInActivity](signinactivity.md) | Get the last signed-in date and request ID of the sign-in for a given user. Read-only.<br><br>Returned only on `$select`. Supports `$filter` (`eq`, `ne`, `not`, `ge`, `le`) *but* not with any other filterable properties. <br><br>**Note:** <br/><li> Details for this property require an Azure AD Premium P1/P2 license and the **AuditLog.Read.All** permission.<li>This property is not returned for a user who has never signed in or last signed in before April 2020.|
```

File: api-reference/v1.0/resources/user.md [hunk 2]
```diff
@@ -204,7 +204,7 @@ This resource supports:
...
+|signInActivity | [signInActivity](signinactivity.md) | Get the last signed-in date and request ID of the sign-in for a given user. Read-only.<br><br>Returned only on `$select`. Supports `$filter` (`eq`, `ne`, `not`, `ge`, `le`) *but not with any other filterable properties*. <br><br>**Note:** <br/><li>Details for this property require an Azure AD Premium P1/P2 license and the **AuditLog.Read.All** permission.<li>You must specify `$select=signInActivity` or `$filter=signInActivity` while [listing users](../api/user-list.md), as the signInActivity property is not returned by default.<li>This property is not returned for a user who has never signed in or last signed in before April 2020.|
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/user.md:207]:
```suggestion
|signInActivity | [signInActivity](signinactivity.md) | Get the last signed-in date and request ID of the sign-in for a given user. Read-only.<br><br>Returned only on `$select`. Supports `$filter` (`eq`, `ne`, `not`, `ge`, `le`) *but not with any other filterable properties*. <br><br>**Note:** <br/><li>Details for this property require an Azure AD Premium P1/P2 license and the **AuditLog.Read.All** permission.<li>This property is not returned for a user who has never signed in or last signed in before April 2020.|
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you @shlipsey3 for your work on this PR. Approved with suggestions.

**Review by FaithOmbongi — APPROVED** [review]
Thank you @shlipsey3 for your work on this PR. Approved with suggestions.

**Pull request merged**